frontend/src/lib/components/progressBar/ProgressBar.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+	import { tweened } from 'svelte/motion'
+	import { cubicOut } from 'svelte/easing'
+	import { isLoop, type Progress } from './model'
+	import ProgressBarGeneralPart from './ProgressBarGeneralPart.svelte'
+	import ProgressBarLoopPart from './ProgressBarLoopPart.svelte'
+
+	export let steps: Progress = []
+	export let startIndex = 0
+	const length = 100 / steps.length
+	const percent = tweened(+(length * startIndex).toFixed(0), {
```

Comment:
Is the leading + usefull ? Everything seems to be numbers

Reply:
It's needed because `toFixed()` returns a string